package.json

```diff
@@ -16,7 +16,7 @@
     "email": "nicholls@ca.ibm.com"
   },
   "engines": {
-    "vscode": "^1.28.2"
+    "vscode": "^1.30.2"
```

Comment:
@petenicholls I had to update the minimum required version of VS Code in order to get the TypeScript compilation to work. [V1.28.2](https://code.visualstudio.com/updates/v1_28) is from September 2018. [V1.30.2](https://code.visualstudio.com/updates/v1_30) is from November 2018. I think that is reasonable? If this is okay, with you, please approve and then I'll merge this pull request in.